nRF Cloud FSM cleanup redux

plskeggs opened the pull request:
Removed left-behind CLOUD_UA_BUTTONS  and CLOUD_UA_CONSOLE.
Fixed text for CLOUD_BUTTON_INPUT to be more meaningful and not identical to CLOUD_BUTTON.
Remove obsolete reference to CLOUD_UA_CONSOLE in lte_ble_gateway sample.

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/NordicPlayground/fw-nrfconnect-nrf?pullRequest=1827) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/NordicPlayground/fw-nrfconnect-nrf?pullRequest=1827) before we can accept your contribution.<br/><hr/>**Pete Skeggs** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/NordicPlayground/fw-nrfconnect-nrf?pullRequest=1827) it.</sub>

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/NordicPlayground/fw-nrfconnect-nrf?pullRequest=1827) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/NordicPlayground/fw-nrfconnect-nrf?pullRequest=1827) before we can accept your contribution.<br/><hr/>**Pete Skeggs** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/NordicPlayground/fw-nrfconnect-nrf?pullRequest=1827) it.</sub>

rlubos commented:
@thst-nordic Could you please add @plskeggs to an appropriate group, he's not recognized as a Nordic employee.

NordicBuilder commented:
**Some checks failed. Please fix and resubmit.**

## Nits issues

```

Redundant 'source "$(ZEPHYR_BASE)/Kconfig.zephyr" in 'applications/asset_tracker/Kconfig'. Just do 'source "Kconfig.zephyr"'
instead. The $srctree environment variable already points to the Zephyr root,
and all 'source's are relative to it.

Missing newline at end of 'applications/asset_tracker/Kconfig'. Check your text editor settings.
```



*Tip: The bot edits this comment instead of posting a new one, so you can check the comment's history to see earlier messages.*

plskeggs commented:
This PR is failing on Nits related to Kconfig.  I don't think I introduced the problems.  One was a missing newline on the final line of the file, and the other is on a redundant source $(ZEPHYR_BASE)/KConfig.zephyr.

I went ahead and added the newline on the end of the file, but the redundant source issue is still there.  @mbolivar suggested that perhaps is actually being caused by a CI issue as it is having a lot of trouble today on other PRs.  So I will wait until Monday and see where things stand.
